ui/js/directives/treeherder/clonejobs.js

```diff
@@ -274,10 +274,9 @@ treeherder.directive('thCloneJobs', [
                     ev.preventDefault();
                     //Middle mouse button pressed
                     ThJobModel.get(this.repoName, job.id).then(function(data){
-                        //Retrieve the job reference data and open a new
-                        //window on the job's log file
-                        if(data.logs.length > 0){
-                            window.open(data.logs[0].url, "Log");
+                        // Open the logviewer in a new window
+                        if (data.logs.length > 0 && data.state === 'completed') {
```

Comment:
If we know that a completed state is implicit when `.logs.length` is positive, I'll remove the extra `.state` check. I think it's my paranoia with the current state of some log artifacts :)

I was going to test against the log parsing status (eg. "in progress"), but that object wasn't immediately available as far as I could tell. I think it's pretty unlikely to click on a job with a fleeting 'parsing' state, which I think would be the only reason for using it.